src/include/storage/write_ahead_log/log_manager.h

```diff
@@ -69,14 +69,14 @@ class LogManager {
 
  private:
   // TODO(Tianyu): This can be changed later to be include things that are not necessarily backed by a disk
-  // (e.g. logs can be streamed out to the network for remote replication)
+  //  (e.g. logs can be streamed out to the network for remote replication)
```

Comment:
nit: these look like unintentional changes

Reply:
Oh, I think I may have done this so that CLion highlights multiline todos: https://blog.jetbrains.com/clion/2018/10/clion-2018-3-eap-profiler-multiline-todo/. I don't feel strongly either way (since it might look weird for anyone not using CLion), but it is nice to have the entire TODO highlighted consistently.